sdk/formrecognizer/azure-ai-formrecognizer/tests/preparers.py

```diff
@@ -18,7 +18,7 @@
 FormRecognizerPreparer = functools.partial(
     PowerShellPreparer,
     'formrecognizer',
-    formrecognizer_test_endpoint="https://region.api.cognitive.microsoft.com/",
+    formrecognizer_test_endpoint="https://fakeendpoint.cognitiveservices.azure.com/",
```

Comment:
also remove the trailing slash here so it matches the conftest sanitizer value. In fact, I think PowershellPreparer sanitizing is now enabled so you might not even need the conftest sanitizer anymore. Haven't tested that yet myself.